setup.py

```diff
@@ -63,6 +65,9 @@
     author_email="snakecharmers@ethereum.org",
     url="https://github.com/ethereum/eth-hash",
     include_package_data=True,
+    install_requires=[
```

Comment:
I just noticed this - Looks like we dropped the eth-utils dependency here: https://github.com/ethereum/eth-hash/pull/33

Reply:
noted, removed.